### DISCUSSION TITLE
Transferencia errónea

### DISCUSSION THREAD

**Discussion by Amcl123** [blockscout/blockscout#11343] [Q&A]
Pase mis monedas de World a binance pero me sale transferencia errónea, y binance no me permite recuperarlas, como podría hacerlo desde Worldcoin y poder recuperar mis monedas

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
